docs/_sass/_main.scss

```diff
@@ -520,7 +633,93 @@ button {
         }
     }
 
+    p.description,
+    p.url {
+        margin: 0;
+        font-weight: normal;
+    }
+
+    p.description {
+        padding: 20px 0 20px 0;
+    }
+
+    p.url {
+        padding: 0;
+        font-size: 0.8em;
+        color: $color-gray-label;
+    }
+}
+
+.hub-card {
```

Comment:
Can you please put some work into DRYing up these styles a bit?

- Add the `card` class back with all the styles that are shared between them
- `hub-card` and `platform-card` can just do `@extend .card;`, then add in the only styles that are different between the two.

Reply:
Thanks!